DEV-4/DEV-4/Lecture.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+namespace DEV_4
+{
+    /// <summary>
+    /// Class for lectures.
+    /// </summary>
+    class Lecture : Material
+    {
+        public string textOfTheLecture;
+        public string TextOfTheLecture
+        {
+            get
+            {
+                return textOfTheLecture;
```

Comment:
If the body of the get or set method of a property consists of a single statement, the statement is
written on the same line as the method signature. White space is inserted between the property
method (get, set) and the opening brace. This will create visually more compact class definitions..